validphys2/src/validphys/core.py

```diff
@@ -750,6 +750,15 @@ def __str__(self):
 
 # TODO: Decide if we want methods or properties
 class Stats:
+    """Class holding statistical information about the object used in validphys
+    By the PDF convention, the central value corresponds to member 0.
+    This is equal to the mean of the error members only for the PDF
+    itself and for linear quantities such as DIS-type observables.
+
+    If you want to obtain the average of the central value you can do:
+    ``np.mean(stats_instance.error_members, axis=0)``
```

Comment:
I don't really understand this sentence?

Reply:
```suggestion
    If you want to obtain the average of the error members you can do:
    ``np.mean(stats_instance.error_members, axis=0)``
```